controllers/tasks.js

```diff
@@ -25,10 +29,17 @@ const getTask = async (req, res, next) => {
   // extract userId and spaceId from request query
   const { userId, spaceId } = req.query;
 
+  // extract requested view from request query
+  let { view } = req.query;
+  if (view !== 'compose') {
```

Comment:
Make constants.

Reply:
`view` gets reassigned if it's not `compose` (so has to be `let`)

Reply:
Yes. I was referring to factoring out the `'compose'` string.